src/include/sof/audio/dcblock/dcblock.h

```diff
@@ -11,6 +11,18 @@
 #include <stdint.h>
 #include <sof/platform.h>
 #include <ipc/stream.h>
+#include <sof/compiler_info.h>
+
+/* __XCC__ is both for xt_xcc and xt_clang */
+#if defined(__XCC__)
+#include <xtensa/config/core-isa.h>
+
+#define DCBLOCK_GENERIC 0
+#define DCBLOCK_HIFI4 1
+#else
+#define DCBLOCK_GENERIC 1
+#define DCBLOCK_HIFI4 0
```

Comment:
I think it's the usual and lighter practice is not to define anything and use `#ifdef`

Reply:
thanks